pkg/fission-cli/cmd/mqtrigger/update.go

```diff
@@ -148,6 +149,14 @@ func (opts *UpdateSubCommand) complete(input cli.Input) (err error) {
 }
 
 func (opts *UpdateSubCommand) run(input cli.Input) error {
+	if input.Bool(flagkey.SpecSave) {
+		specFile := fmt.Sprintf("mqtrigger-%v.yaml", opts.trigger.ObjectMeta.Name)
+		err := spec.SpecSaveUpdate(*opts.trigger, specFile)
```

Comment:
We should validate spec before calling SpecSaveUpdate function, same as updating environment.